datastore/longhorn.go

```diff
@@ -225,6 +225,34 @@ func (s *DataStore) syncSettingCRsWithCustomizedDefaultSettings(customizedDefaul
 	return nil
 }
 
+func (s *DataStore) UpdateDefaultSettingConfigMapInstanceManagerImage(imImage string) error {
```

Comment:
ConfigMap is used for setting purpose, so if the setting is removed, should not use the config map for the persistence.

I would suggest making the default instance manager setting as hidden instead of removing it to achieve the same goal w/o touching the setting config map.

Reply:
@innobead
Do we have any example to make a setting as hidden? Just need a reference because I have no idea how to hide a Longhorn setting

Reply:
```go
		Type:     SettingTypeDeprecated,
```

This type is misleading, but it will actually be hidden and you can make it readyonly together.

Reply:
Store the `backing image manager image` and `instance manager image` into the `Datastore` when initializing longhorn-manager daemon